src/handlers/bulkTransition.js

```diff
@@ -70,6 +70,7 @@ export const handler = async (
     });
 
     // Await completion of all promises
+    // CR: Mickey: Promise.all fails if any of its child promises fail.  Is that expected behavior?
```

Comment:
@MickeyMcc This is expected behavior. The promise called has the neverThrowError within it for executing the workers. In spite of that, if an error is thrown, there should be something wrong with the framework which obviously should be fully tested before release of the framework.